### DISCUSSION TITLE
Vista tarea: No está funcionando la configuracion vista por defecto de la sm

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#338] [4. Bug Reports] [answered]
Hola

Estaba configurando algunos filtros de task con vista kanban, pero no lo está tomando.

**Adjunto SS sobre la configuracion de la SM**
![image](https://user-images.githubusercontent.com/61419471/125815938-c79a3a98-4d96-4512-bdc6-e7fefd548088.png)

**Adjunto SS sobre la carga por defecto de la vista task**
![image](https://user-images.githubusercontent.com/61419471/125815995-d2d4273f-4b15-4e00-bd2d-675c3c588478.png)

**Dejo un link de video sobre lo que indico:** https://drive.google.com/file/d/17igsWz7lQyxPXNUwBQ6ZgiAY6nv70GXg/view?usp=sharing

@gmoplaza tu ves esto?

Gracias =>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Gracias por el reporte, estamos al tanto del incidente y esperamos realizar una corrección a la brevedad.
